react/react1/week1/React - ONE/vite-project/src/Card.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
```

Comment:
Perfect. Props used correctly, and awesome to see you do some styling also 🥇

Reply:
@LucasIversen Thank you! I will try to keep this up and explore React more :)